instrumentation/aws-sdk/aws-sdk-2.2/library/build.gradle.kts

```diff
@@ -72,6 +89,7 @@ tasks {
   }
 
   check {
+    dependsOn(testing.suites)
```

Comment:
Note this causes other suites defined above to be run that I think previously weren't